### DISCUSSION TITLE
FPS issue

### DISCUSSION THREAD

**Discussion by Rzk0** [ppy/osu#23502] [Q&A]
so, when i using OpenGL rendering fps is uncap
but when change to dx11, fps stuck on vsync 
![a](https://github.com/ppy/osu/assets/133306493/593fca2e-ac6e-4e51-9ce8-d0721f08ad36)
![b](https://github.com/ppy/osu/assets/133306493/04580e22-255f-46ac-8e6c-9089785f4450)

**Comment by Gyoshi** [comment]
I have the same, but only in windowed & borderless windowed, so not really an issue for me

**Comment by Theighlin** [comment]
Duplicate of #22962 probably
